agent/tool-scripts/base-tool

```diff
@@ -191,9 +191,13 @@ esac
 
 function usage {
 	printf -- "The following options are available:\n\n"
+	printf -- "Operational options - do *not* specify any of these when you register a tool:\n\n"
 	printf -- "\t--install                 verify this perf tool will work\n\n"
 	printf -- "\t--start|stop|postprocess  start/stop/postprocess the data collection\n"
 	printf -- "\t-d str|--dir=str          directory to store data collection (required)\n"
+
+        printf -- "\nTool-specific options - these should be specified when you register a tool:\n\n"
```

Comment:
This line appears to be indented with 8 spaces whereas surrounding lines are indented with a tab rendered by GitHub as 4 spaces. The *file* declares that the two should be rendered identically but clearly GitHub isn't a member of that conspiracy and it'd be nicer if the indentation was consistent in source rather than just in preferred rendering.